### ISSUE TITLE
no method 'setPrompt'

### ISSUE THREAD

**Issue by djensen47** [sidorares/node-vim-debugger#36]
I'm using node 0.10.36 (the current *stable* production version of node).

I receive the following error after I follow the step to start vim with `vim -nb`

```
TypeError: Object #<REPLServer> has no method 'setPrompt'
    at VimConnection.showScript (/Users/djensen/.nvm/v0.10.36/lib/node_modules/vimdebug/lib/vim-connection.js:181:7)
    at /Users/djensen/.nvm/v0.10.36/lib/node_modules/vimdebug/lib/vim-connection.js:130:10
    at Client._onResponse (/Users/djensen/.nvm/v0.10.36/lib/node_modules/vimdebug/node_modules/v8-debugger/lib/client.js:135:5)
    at Protocol.execute (/Users/djensen/.nvm/v0.10.36/lib/node_modules/vimdebug/node_modules/v8-debugger/lib/protocol.js:86:14)
    at Client.<anonymous> (/Users/djensen/.nvm/v0.10.36/lib/node_modules/vimdebug/node_modules/v8-debugger/lib/client.js:49:14)
    at Client.emit (events.js:95:17)
    at Client.<anonymous> (_stream_readable.js:765:14)
    at Client.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at emitReadable (_stream_readable.js:423:5)
```

**Comment by sidorares** [comment]
that was a side effect of fixing iojs / node 0.12 issue #17
needs to be done similarly to https://github.com/sidorares/crconsole/commit/10e204c4664cf4f8587a901094b74865abb950e6
I'll try to do right now and test with both .10 and .12

**Comment by djensen47** [comment]
:+1:  Thanks!

**Comment by sidorares** [comment]
published as v0.2.3, tested with node 0.10.38 and iojs 1.8.1
